crawl-ref/source/dat/descript/features.txt

```diff
@@ -426,14 +431,29 @@ A gateway to the Iron City of Dis
 
 A gateway leading to the Iron City of Dis.
 %%%%
+A gateway leading out of Dis
+
+A gateway leading back to the Vestibule of Hell, and whatever horrors you left
+behind there.
+%%%%
 A gateway to the decaying netherworld of Tartarus
 
 A gateway leading to Tartarus.
 %%%%
+A gateway leading out of Tartarus
+
+A gateway leading back to the Vestibule of Hell, and whatever horrors you left
+behind there.
+%%%%
 A gateway to the freezing wastes of Cocytus
 
 A gateway leading to Cocytus.
 %%%%
+A gateway leading out of Cocytus
+
+A gateway leading back to the Vestibule of Hell, and whatever horrors you left
+behind there.
+%%%%
```

Comment:
Instead of adding the same description four times, you can add it once and then use the `<feature>` syntax for reusing descriptions. For example,

```
A gateway leading out of Dis

<A gateway leading out of Gehenna>
```

Reply:
Thanks; I've changed that now.